addons/l10n_in_website_sale/static/src/js/portal.js

```diff
@@ -0,0 +1,86 @@
+import publicWidget from "@web/legacy/js/public/public_widget";
+import { debounce } from "@web/core/utils/timing";
+
+publicWidget.registry.portalDetailsExtend = publicWidget.registry.portalDetails.extend({
+    events: {
+        'input input[name="vat"]': "_onChangeVat",
+        'change select[name="country_id"]': "_onCountryChange",
+    },
+
+    init: function () {
+        this._super.apply(this, arguments);
+
+        this._onChangeVat = debounce(this._onChangeVat.bind(this), 400);
+        this.countryCode = document.querySelector(
+            'form[action="/my/account"]'
+        ).dataset.companyCountryCode;
+        this.l10n_in_gst_treatment = document.getElementById("l10n_in_gst_treatment_container");
```

Comment:
```suggestion
        this.l10n_in_gst_treatment = this.el.querySelector(".div_l10n_in_gst_treatment_container");
```
lets not access the element from global document instead will access it through the root element, this way we can limit the search.